README.md

```diff
@@ -1,158 +1,53 @@
-# CodersCamp 2020 - Projekt końcowy (FullStack — Node.js + React)
-**CodersCamp (coderscamp.edu.pl) - Największy otwarty kurs programowania webowego** 
+# Dance School Management System
 
-### Wprowadzenie
-Gratulacje! 
-Udało Wam się dotrzeć prawie do końca kursu. 
-Posiadacie już wiele umiejętności, które są konieczne, aby rozpocząć karierę programisty.
-W trakcie tego projektu powinniście dowieść, że jesteście gotowi wykonywać kompletne aplikacje webowe.
-Wraz ze zbieraniem wymagań, dzieleniem pracy, zarządzaniem, całym procesem wytwórczym, testowaniem oraz wdrażaniem.
+### Informacje:
 
-### Zasady wykonywania projektu (wspólne dla wszystkich grup i mentorów): 
+Projekt zespołowy wykonany w ramach kursu CodersCamp 2020
 
-##### W projekcie każdy z uczestników powinien zaprezentować praktyczną znajomość poniższych zagadnień związanych z Node.js i React:
+- Uczestnicy projektu:
+- Marlena - TechLead
+- Radek - Development Manager
+- Joanna - Product Owner
+- Maciej
+- Dariusz
+- Adam
 
-**Node.js / Express**
+### Cel projektu:
+
+Celem projektu było napisanie aplikacji wykorzystującej dotychczas nabytą wiedzę z nodejs oraz baz danych.
+
+### Zasady wykonywania projektu:
+
+- Node.js / Express
```

Comment:
Express bym usunęła, a w ogóle to cały ten jeden podpunkt, bo już jest niżej w użytych technologiach, wydaję mi się że w jednym miejscu wystarczy o tym wspomnieć, jak myślisz?